<?xml version="1.0"?>
<launch>
    <arg name="namespace" default="intel_aero" />
    <node name="vision" pkg="img_center_alignment" type="dronet_node.py" output="screen">
        <remap from="camera" to="$(arg namespace)/image_compressed"/>
        <param name="json_model_path" value="$(find img_center_alignment)/models/model_struct.json"/>
        <param name="weights_path" value="$(find img_center_alignment)/models/model_weights_59.h5"/>
        <param name="onboard_images_folder" value="$(find img_center_alignment)/onboard_images"/>
        <param name="target_size" value="340,255"/>
  </node>
</launch>

